aries_cloudagent/resolver/__init__.py

```diff
@@ -1,5 +1,6 @@
 """Interfaces and base classes for DID Resolution."""
 
+from aries_cloudagent.ledger.base import BaseLedger
```

Comment:
This should be a relative import with the others

Reply:
Ah, woops. Auto-import... thanks for the catch.